srcs/expansion/expansion_utils.c

```diff
@@ -89,7 +89,7 @@ static int	needs_escaping(char c, int state)
 	Escape double-quotes, dollar-signs and backslashes.
 	A backslash is placed before each of these characters.
 
-	Result is heap-allocated.
+	Result is malloc()-ed.
```

Comment:
` x_malloc()-ed` but that's just being a tad retentive.

Reply:
with also modify the size to use `sizeof(char)` so it's inline with the previouslt used style.

Reply:
modified with [605d97f](https://github.com/sophiakoulen/minishell/pull/98/commits/605d97fc84a44c33b3538d434b4b489aba632b78) commit